src/modules/logger/logged_topics.cpp

```diff
@@ -51,6 +51,7 @@ void LoggedTopics::add_default_topics()
 	add_topic("airspeed", 1000);
 	add_optional_topic("airspeed_validated", 200);
 	add_optional_topic("autotune_attitude_control_status", 100);
+	add_topic_multi("battery_info", 5000, 2);
```

Comment:
Could even be much higher, no?

Reply:
Could yes, I thought it's no burden at this low rate. We could theoretically do something more clever and only publish once per log and upon change but our existing mechanisms don't allow that and 5 seconds sample rate is the slowest topic int he list. Should we do every 10 seconds? Or what would you suggest? A relatively short log should contain a sample but that's currently it since the serial number only changes if you swap battery but that usually happens while disarmed.